app/containers/transfer/YoroiPlatePage.js

```diff
@@ -0,0 +1,90 @@
+// @flow
+import React, { Component } from 'react';
+import { observer } from 'mobx-react';
+import globalMessages from '../../i18n/global-messages';
+import WalletRestoreVerifyDialog from '../../components/wallet/WalletRestoreVerifyDialog';
+import type { WalletAccountNumberPlate } from '../../api/ada/lib/storage/models/PublicDeriver/interfaces';
+import type { InjectedProps } from '../../types/injectedPropsType';
+import type { ExplorerType } from '../../domain/Explorer';
+import config from '../../config';
+import {
+  generateStandardPlate,
+} from '../../api/ada/lib/cardanoCrypto/plate';
+
+import type {
+  ConfigType,
+} from '../../../config/config-types';
+
+declare var CONFIG : ConfigType;
+const protocolMagic = CONFIG.network.protocolMagic;
+
+type Props = {|
+  ...InjectedProps,
+  +onNext: void => void,
+  +selectedExplorer: ExplorerType,
+  +onCancel: void => void,
```

Comment:
note: this is a `cancel` and not a back button so unfortunately if you realize you made a mistake in your mnemonic you have to re-enter it from scratch. It was easier to program this way but we can change it in the future if we want.